src/mono/wasm/host/JSEngineHost.cs

```diff
@@ -62,6 +62,8 @@ private async Task<int> RunAsync()
         {
             // v8 needs this flag to enable WASM support
             args.Add("--expose_wasm");
+            // treat file as ES6 module
+            args.Add("--module");
```

Comment:
This is for `dotnet run`, what if user's main JS is not module? The template still has `.mjs` which handles that, right?

Reply:
I'm not sure, I will revert it because it's not necessary.